js/interface.js

```diff
@@ -2289,22 +2287,26 @@ $('#show-access-rules').click(function() {
   });
 });
 
+/**
+ * Check if security rule is present in main security rule list
+ * @returns {Boolean} security rule presence
+ */
 function getSecurityRule() {
-  var flag = false;
+  var isSecurityRule = false;
```

Comment:
`isSecurityRule` suggest it also "isn't" a security rule at times? Perhaps you meant something like `hasSecurityRule` or a simple `found`?

Reply:
did the changes as per your suggestion and used `hasSecurityRule` varible name